### PR TITLE
Activity Submission Page Cleanup

### LINKED ISSUE #75: Update the tooltip for "weight examples" on submit activity page
Image

(on dev ready section in Figma)

also, only display error messages after user has typed, not initially

### PR DESCRIPTION
**Pull request by maxpinheiro**
Closes #75 

## Description

Variety of cleanup to the Activity Submission page:
- replace "Weight Examples" tooltip with new tooltip, and modify `InfoTooltip` to allow messages on the right side
- remove "Year" input and automatically set year
- cleanup form logic with Redux
- update error logic: rather than displaying errors immediately and disabling Submit button, allows users to attempt submit then displays errors (same as user onboarding)

## Screenshots if Applicable

Before:
Screenshot 2023-11-05 at 12 59 51 PM

Weight Tooltip:
weight tooltip

Form Errors:
form errors

Other Description Error (when Other checked but no description present):
other description error

## Checklist
- [x] Ticket number in PR title
- [x] Add ticket number to ("Closes #")
- [x] Move status to Code Review in GH Board
- [ ] People added to reviewers
- [ ] Asked for Review in Slack

### CODE REVIEW

File: src/pages/auth/signin.tsx [hunk 1]
```diff
@@ -12,32 +12,36 @@ export default function SignIn({
   providers,
 }: InferGetServerSidePropsType<typeof getServerSideProps>) {
   return (
-    <div className="w-full flex flex-grow justify-center items-center">
```
Comment (maxpinheiro) [inline src/pages/auth/signin.tsx:15]: just some unrelated formatting that prettier complained about

**Review by Suraj-Ram — APPROVED** [review]
Nice work, LGTM!
